docs/installation.md

```diff
@@ -0,0 +1,6 @@
+Installation
+============
+
+``` {.sourceCode .bash}
```

Comment:
This is being output as:

`{.sourceCode .bash} pip install bulwark`

<img width="1138" alt="Screen Shot 2019-07-28 at 11 53 48 AM" src="https://user-images.githubusercontent.com/19669890/62009547-81743780-b12e-11e9-9c8c-f878403e63de.png">

Reply:
Also a pandoc error